app/views/users/_account.html.erb

```diff
@@ -7,7 +7,7 @@
 <div class="crayons-card mb-6 p-6 grid gap-6">
   <header>
     <h2 class="mb-2"><%= community_name %> API Keys</h2>
-    <p>You can generate personal API keys to use for authentication with the <%= community_name %> API. The API is still in early beta though. The <a href="https://docs.dev.to/api"><%= community_name %> API documentation</a> contains further information.</a></p>
+    <p>You can generate personal API keys to use for authentication with the API. <%= community_name %> is built on the Forem public API (beta). The <a href="https://docs.forem.com/api">documentation</a> contains further information.</a></p>
```

Comment:
```suggestion
    <p>You can generate personal API keys to use for authentication with the <%= community_name %> API.  The API is still in its beta stage. The <a href="https://docs.forem.com/api">documentation</a> contains further information.</a></p>
```

I think we should keep the former wording. The new wording makes it sound like there is a central Forem API, when in reality, each community does have its own API for its own members and data. Let me know what you think!

Reply:
That's true @mstruve! I'll push the changes asap.